src/ui/components/Header/UserOptions.js

```diff
@@ -36,6 +36,14 @@ function UserOptions({ recordingId, setModal }) {
     }
     window.location = dashboardUrl;
   };
+  const onDocsClick = () => {
+    const docsUrl = `https://www.notion.so/Docs-56758667f53a4d51b7c6fc7a641adb02`;
+
+    if (event.metaKey) {
+      return window.open(docsUrl, "replaydocs");
+    }
+    window.open("https://www.notion.so/Docs-56758667f53a4d51b7c6fc7a641adb02", "replaydocs");
```

Comment:
isn't this the same: `window.open(docsUrl, "replaydocs");`

Reply:
what does `"replaydocs"` do? Do you need "_blank"

Reply:
It's totally the same, I'm not sure why I did that. Will change.

_blank causes trouble in this scenario because it means each time you click you're going to load a new tab. By naming it, it means each time you click docs it re-uses the tab. _blank is better for general web, named windows are better for software UI like this.